client/lib/analytics/ad-tracking/record-order.js

```diff
@@ -560,6 +563,50 @@ function recordOrderInJetpackGA( cart, orderId, wpcomJetpackCartInfo ) {
 	}
 }
 
+/**
+ * Records an order in the Akismet.com GA4 Property
+ *
+ * @param {Object} cart - cart as `ResponseCart` object
+ * @param {number} orderId - the order id
+ * @param {Object} wpcomJetpackCartInfo - info about WPCOM, Jetpack, and Akismet in the cart
+ * @returns {void}
+ */
+function recordOrderInAkismetGA( cart, orderId, wpcomJetpackCartInfo ) {
+	if ( ! mayWeTrackByTracker( 'ga' ) ) {
+		return;
+	}
+
+	if ( wpcomJetpackCartInfo.containsAkismetProducts ) {
+		fireEcommercePurchaseGA4(
+			cartToGaPurchase( orderId, cart, wpcomJetpackCartInfo ),
+			Ga4PropertyGtag.AKISMET
+		);
+
+		const akismetParams = [
+			'event',
+			'purchase',
+			{
+				send_to: TRACKING_IDS.akismetGoogleAnalyticsGtag,
+				value: wpcomJetpackCartInfo.akismetCostUSD,
+				currency: 'USD',
+				transaction_id: orderId,
+				coupon: cart.coupon?.toString() ?? '',
+				items: wpcomJetpackCartInfo.akismetProducts.map(
+					( { product_id, product_name_en, cost, volume } ) => ( {
+						id: product_id.toString(),
+						name: product_name_en.toString(),
+						quantity: parseInt( volume ),
+						price: ( costToUSD( cost, cart.currency ) ?? '' ).toString(),
+						brand: GA_PRODUCT_BRAND_JETPACK,
```

Comment:
For brand, it is showing WordPress.com for me
![image](https://user-images.githubusercontent.com/65001528/233425084-2c091c51-a8ee-496b-9b13-e5084c4c8e1a.png)

Is this expected? Should it be Akismet.com or something similar?

Reply:
Yeah, I updated it to Akismet!